extensions/noteplan-3/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "noteplan-3",
+  "title": "NotePlan 3",
+  "description": "Allows you to see your daily plan, search notes and launch straight into NotePlan.",
+  "icon": "command-icon.png",
+  "author": "shakedlokits",
+  "categories": [
+    "Productivity",
+    "Applications",
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search",
+      "title": "Search Notes",
+      "description": "Search all notes",
+      "subtitle": "NotePlan 3",
+      "mode": "view"
+    },
+    {
+      "name": "today",
+      "title": "Daily Plan",
+      "description": "Show today's note",
+      "keywords": ["today"],
+      "subtitle": "NotePlan 3",
+      "mode": "view"
```

Comment:
Cool idea to have a command for the Daily Plan!

Reply:
Sounds cool I'll add it 👌🏽